### PR TITLE
ActivitiesPanel

### PR DESCRIPTION
**Pull request by 06hpathak**
Please ignore some fields like edit, delete and view since they are yeh to be updated based on activities page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@testing-library/jest-dom": "^6.4.0",
     "@testing-library/react": "^14.2.0",
     "@testing-library/user-event": "^14.5.2",
+    "antd": "^5.16.4",
```
Comment (thesiriusacademy) [inline package.json:22]: why this is added?

Comment (06hpathak) [reply package.json:22]: for radio button

Comment (06hpathak) [reply package.json:22]: the radio button we have used in the progress type is provided by ant design

**Review by thesiriusacademy — COMMENTED** [review]
one comment

**Review by thesiriusacademy — APPROVED** [review]

**Pull request merged**